.github/workflows/tests.yml

```diff
@@ -121,6 +121,11 @@ jobs:
             echo PYTEST_MARKERS=${PYTEST_MARKERS} >> $GITHUB_ENV
           fi
 
+      - name: Google auth
+        uses: "google-github-actions/auth@v2"
+        with:
+          credentials_json: "${{ secrets.GCP_CREDENTIALS }}"
+
```

Comment:
Is this step necessary? If I had to guess, it should be enough to pass the credentials to the `GCSFileSystem` in the fixture.

Reply:
Thanks for the explanation!